### PR TITLE
Update golang/x/net dependency to release-branch.go1.11

### PR DESCRIPTION
**Pull request by PrasadG193**
Fixes: https://github.com/kubernetes-sigs/kustomize/issues/823

### CODE REVIEW

**Comment by PrasadG193** [comment]
cc: @wenjiaswe

File: Gopkg.lock [hunk 1]
```diff
@@ -360,12 +360,61 @@
   pruneopts = "NUT"
   revision = "b3f03f55328800731ce03a164b80973014ecd455"
 
+[[projects]]
+  digest = "1:c19e61cad0c7bc59584f69c4b4c036b8271fcb48a1cdf693c5cb36efd5b537ac"
+  name = "sigs.k8s.io/kustomize"
```
Comment (Liujingfang1) [inline Gopkg.lock]: Why `kustomize` is added to this file?

Comment (PrasadG193) [reply Gopkg.lock]: Looks like I cloned the repo in the wrong path. Let me fix this

**Pull request merged**